scripts/build-android.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -euf
+
+TARGET=${1:-debug}
+
+_current_dir=$(cd "${BASH_SOURCE%/*}" && pwd)
```

Comment:
Can we call this CURRENT_DIR?

Reply:
I copied this from https://github.com/status-im/status-react/blob/develop/scripts/setup#L10
Right now our scripts are a mix of convention. Happy to use whichever.

Reply:
Ok I missed that one, seems like we mix styles a bunch. I'm used to seeing variables like be uppercase but no big deal.

Since we are starting to have a bunch of shell scripts, it might be a good idea to adopt some standard shell style guideline. For example this one https://google.github.io/styleguide/shell.xml (it recommends lowercase variables https://google.github.io/styleguide/shell.xml#Variable_Names actually).

Reply:
Found this: https://www.chromium.org/chromium-os/shell-style-guidelines where they say:

> There are two standards that are permissible: GNU Bash and POSIX shell.  When writing code that is used on developer systems or dev/test Chromium OS images, always use bash.  For scripts that are used on the release Chromium OS image, you should be using POSIX shell.

which makes sense. Something like this would help us get rid of inconsistencies and cut down on style comments in PRs.

Oh well, a bit out of scope of this PR :P